common/src/Assets/AttributeDefinition.h

```diff
@@ -77,11 +76,7 @@ namespace TrenchBroom {
             }
 
             const T& defaultValue() const {
-                if (!hasDefaultValue()) {
```

Comment:
calling operator* on an empty std::optional is undefined behaviour, so we should keep an `ensure()` here.

Reply:
That's a good alternative, yes.

Reply:
Done.